README.md

```diff
@@ -1,25 +1,30 @@
-# Terragrunt Atlantis Config
+<p align="center">
+  <img alt="Terragrunt Atlantis Config by Transcend" src="https://user-images.githubusercontent.com/7354176/78756035-f9863480-792e-11ea-96d3-d4ffe50e0269.png"/>
+</p>
+<h1 align="center">Terragrunt Atlantis Config</h1>
+<p align="center">
+  <strong>Generate Atlantis Config for Terragrunt projects.</strong>
+</p>
+<br />
 
-Generates Atlantis Config for Terragrunt projects"
+## What is this
 
-## What is this?
+[Atlantis](runatlantis.io) is an awesome tool for Terraform pull request automation. Each repo can have a YAML config file that defines Terraform module dependendcies, so that PRs that affect dependent modules will automatically generate `terraform plan`s for those modules.
 
-Atlantis (runatlantis.io) is an awesome tool for automating terraform pull request automation. Each repo can define a yaml config file that specifies which terraform modules depend on other modules so that PRs that affect dependent modules will also make `terraform plan`s appear for the depending modules.
+[Terragrunt](https://terragrunt.gruntwork.io) is a Terraform wrapper, which has the concept of dependencies built in to its configuration.
 
-Bring in terragrunt (https://terragrunt.gruntwork.io/), a thin terraform wrapper that has a concept of dependencies built into its configuration.
-
-This tool creates yaml for terragrunt projects by:
+This tool creates Atlantis YAML configurations for Terragrunt projects by:
 
 - Finding all `terragrunt.hcl` in a repo
-- Evaluating their dependency and terraform source blocks to find their dependencies
+- Evaluating their dependency and Terraform source blocks to find their dependencies
```

Comment:
This doesn't read well to me but I don't have enough context to revise it

Reply:
ah, "dependency" and "terraform" are keywords in the HCL schema used with terragrunt. So we should maybe leave the "t" as lowercase, but wrap these two words in backticks